modules/ebs_snapshot_explorer/main.py

```diff
@@ -1,26 +1,38 @@
 #!/usr/bin/env python3
 """Module for ebs_snapshot_explorer"""
+import argparse
 from copy import deepcopy
 import json
 from pathlib import Path
 
 from botocore.exceptions import ClientError
 
-from . import parser
-from . import module_info
-
-
-def input_helper(input):
-    """Helper function that loops until a successful response is given"""
-    prompt = '    Load next set of volumes? (y/n) '
-    while True:
-        response = input(prompt)
-        if response.lower() == 'y':
-            return True
-        elif response.lower() == 'n':
-            return False
-
-def load_volumes(client, print, input, instance_id, volume_ids):
+module_info = {
+    'name': 'ebs_snapshot_explorer',
+    'author': 'Alexander Morgenstern alexander.morgenstern@rhinosecuritylabs.com',
+    'category': 'post_exploitation',
+    'one_liner': 'Loads EBS snapshots and volumes so they are easily accessible.',
```

Comment:
I think something like: "Restores and attaches EBS volumes/snapshots to an EC2 instance of your choice."